### ISSUE TITLE
Bug in use of rooms variable / access control to bot

### ISSUE THREAD

**Issue by chrisfahr** [shawnanastasio/python-matrix-bot-api#8]
Hi,
there is an issue with your usage of the rooms variable.
In matrix_bot_api.py line 43 (else branch) you try to use it like an array of room objects.
This fails in line 46 when you try to assign handlers.
In the other branch you store (string) ids in it. Then again in line 71 you store objects.
This prevents one from using rooms in the constructor to limit access to the bot.

**Comment by shawnanastasio** [comment]
Good catch. I suppose I'll update all branches to treat them as an array of string room ids, unless you think room objects would be better?

**Comment by chrisfahr** [comment]
I think using strings is fine.
Otherwise an user of the libary would have to call the underlying API to get the room objects,
which kind of contradicts the use case he has in the first place.
